src/components/main.vue

```diff
@@ -4,10 +4,10 @@
     <LocationProvider location="results">
       <Results />
     </LocationProvider>
-    <LocationProvider v-if="showRelatedPrompts" location="related_prompts">
+    <LocationProvider v-if="showRelatedPrompts" :location="semanticsLocation">
```

Comment:
Is this intentional? Is it correct?

Reply:
Yep, related_prompts should be added as a location value in x-components if we want to use it here,
maybe it should also be discussed with the team @victorcg88